python/paddle/v2/framework/tests/op_test_util.py

```diff
@@ -61,10 +61,7 @@ def test_all(self):
                 for out_name in func.all_output_args:
                     actual = numpy.array(scope.find_var(out_name).get_tensor())
                     expect = getattr(self, out_name)
-                    # TODO(qijun) The default decimal is 7, but numpy.dot and eigen.mul
-                    # has some diff, and could not pass unittest. So I set decimal 3 here.
-                    # And I will check this in future.
-                    numpy.testing.assert_almost_equal(actual, expect, decimal=3)
+                    numpy.isclose(actual, expect)
```

Comment:
It seems that `numpy.isclose` compares the relative difference, and `numpy.testing.assert_almost_equal` compares the absolute difference. Why should we use relative other than absolute difference here?